interactive-bots/helloworld/sColin16/notes.md

```diff
@@ -0,0 +1,20 @@
+Problems I faced:
+=================
+
+1. **Obtaining the zuliprc configuration on the dev enviornment** - Although I was able to download the file onto my local file system, I was 
+unable to use the configuration file in the dev enviornment, because it was not there. To solve this problem, I used the scp command from my 
+local enviornment, to send the configuration file to the development enviornment.
+
+2. **Running the bot from a seperate console** - Initially, I attempted to run the bot without the zulip test instance running. Of course, the 
```

Comment:
We usually call the Zulip test instance the "Zulip development environment", or simply "Zulip dev env". Can you use that wording?